index.bs

```diff
@@ -3497,6 +3652,179 @@ partial dictionary MLOpSupportLimits {
     </div>
 </details>
 
+<div class="note">
+  <details open>
+    <summary>
+    The behavior of the {{MLGraphBuilder/sign()}} operation can be [EMULATED]
+    </summary>
+    <pre highlight="js">
+    function sign(builder, input, options) {
+      let zero = builder.constant(input.dataType, 0);
+      let positiveOne = builder.constant(input.dataType, 1);
+      let negativeOne = builder.constant(input.dataType, -1);
+
+      return builder.where(
+        builder.greater(input, zero),
+        positiveOne,
+        builder.where(
+          builder.lesser(input, zero),
+          negativeOne,
+          positiveOne));
+    }
+    </pre>
+  </details>
+</div>
+
+
+### dequantizeLinear ### {#api-mlgraphbuilder-dequantizelinear}
+Dequantizes an integer tensor to floating point space using the scale and zero-point bias, where `output = (input - zeroPoint) * scale`.
+
+The operation will be [=broadcast=] according to [[!numpy-broadcasting-rule]]. The input tensors must be [=bidirectionally broadcastable=]. The [=MLOperand/rank=] of the output tensor is the maximum [=MLOperand/rank=] of the input tensors. For each dimension of the output tensor, its size is the maximum size along that dimension of the input tensors, and each dimension must be blockwise compatible with the output (e.g. given an input shape [12], scales of the following shapes are blockwise compatible {[1], [3], [4], [6], [12]} as they are all multiples of the input dimensions, but a shape of [5] would not be).
```

Comment:
Maybe introduce "blockwise compatible" as a definition, to avoid duplicating it between dequantizeLinear() and quantizeLinear()?

Reply:
⌛